src/bin/am/main.rs

```diff
@@ -27,6 +28,8 @@ async fn main() {
         std::process::exit(1);
     }
 
+    tokio::task::spawn(updater::update_check());
```

Comment:
Won't tokio just stop this task once it reaches the end of the `main` function? So if the command that was run finishes quicker than the update check, then the update check will only be run partially.

Reply:
how would i prevent that? join it?

Reply:
The spawn returns a handle, and you can await that handle at the end. We do a similiar thing with fp: https://github.com/fiberplane/fp/blob/main/src/main.rs#L327 (fp also does a timeout to ensure the cli doesn't hang).